### ISSUE TITLE
Plotting colorbar

### ISSUE THREAD

**Issue by theogf** [stelmo/TernaryDiagrams.jl#15]
It does not seem possible to plot a colorbar at the moment.
Doing:
```
p = ternarycontourf!(ax, a1, a2, a3, ws)
Colorbar(fig[1,1][1,2], p)
```
I get the following error:
```
KeyError: key :colorrange not found
```

**Comment by stelmo** [comment]
Yeah, you need to manually set the colorbar limits, it is not automatically inferred :/

**Comment by theogf** [comment]
I am not sure but I think that by having a colorrange (and possibly more attributes in the `TernaryContour(f)` objects one can get the conversion for free?

**Comment by stelmo** [comment]
I think you are right, unfortunately I don't have time to really delve into it at the moment. Maybe in a month or two I can look

**Comment by theogf** [comment]
I can try to make a PR
